pages/dkp/kommander/2.1/release-notes/2.1.1/index.md

```diff
@@ -77,6 +77,93 @@ The following services and service components have been upgraded to the listed v
 - traefik-forward-auth: 0.3.2
 - velero: 3.1.3
 
+## Known Issues 
+
+The following items are known issues with this release.
+
+### Create cert-manager resources on attached clusters with cert-manager pre-installed
+
+If you attach a cluster that already has `cert-manager` installed, you need to manually create the three cert-manager resources, in the yaml file, after attaching your cluster.
+
+For example, [Konvoy-created clusters that are self-managed][konvoy-self-managed] have `cert-manager` already installed to the `cert-manager` namespace.
+
+Verify that your cluster is self-managed by:
+
+```bash
+export KUBECONFIG=<kubeconfig-path>
+export WORKSPACE_NAMESPACE=<your-workspace-namespace>
+kubectl get kommandercluster -n $WORKSPACE_NAMESPACE
+```
+
+If your cluster is not self-managed, the output resembles this example:
+
+```bash
+error: the server doesn't have a resource type "kommandercluster"
+```
+
+If your cluster is self-managed, the output resembles this example:
+
+```bash
+NAME     AGE   PHASE
+my-cluster   30m   Provisioned
+```
+
+If your cluster is self-managed, then create the following yaml file:
+
+```yaml
+cat << EOF > cert_manager_root-ca.yaml
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: kommander-bootstrap-ca-issuer
+  namespace: $WORKSPACE_NAMESPACE
+spec:
+  selfSigned: {}
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: kommander-bootstrap-root-certificate
+  namespace: $WORKSPACE_NAMESPACE
+spec:
+  commonName: ca.kommander-bootstrap
+  dnsNames:
+    - ca.kommander-bootstrap
+  duration: 8760h
+  isCA: true
+  issuerRef:
+    name: kommander-bootstrap-ca-issuer
+  secretName: kommander-bootstrap-root-ca
+  subject:
+    organizations:
+      - cert-manager
+---
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: kommander-bootstrap-issuer
+  namespace: $WORKSPACE_NAMESPACE
+spec:
+  ca:
+    secretName: kommander-bootstrap-root-ca
+EOF
+```
+
+Next, apply this file to your cluster:
+
+```sh
+kubectl apply -f cert_manager_root-ca.yaml
+```
+
+Finally, fix the broken certificates for the attached cluster:
```

Comment:
does this need to be run on all attached clusters?

Reply:
this is the common step for both self-managed / non self-managed attached clusters. and yes, needs to be run on all attached clusters.

Reply:
note instead of checking "self-managed", we gonna only care about whether cert-manager is pre installed